.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will test this Node.js package.
+# For more information see: https://docs.github.com/en/actions/guides/building-and-testing-nodejs
+
+name: Test with NPM
```

Comment:
Just "NPM", then? Or maybe "CI"? I like "Build and test", since it summarizes what we're doing here, or "Test with NPM", since it makes it clear how we're using this tool. But I'm happy with anything here, so feel free to suggest whatever you think is best.

Reply:
> I like "Build and test", since it summarizes what we're doing here

Yeah, so let's use that. Just use Title Case. ("Build and Test")

Reply:
Fixed in 611d0b3.